test/writer.test.js

```diff
@@ -144,6 +144,28 @@ describe('writer', function () {
       });
     });
 
+    it('ommits commit category if there was no category defined', function () {
+      var hash = '1234567';
+      var commits = [
+        { type: 'feat', category: 'testing', subject: 'did some testing', hash: hash + 'a' },
+        { type: 'chore', category: '', subject: 'other changes', hash: hash + 'a' }
+      ];
+
+      return Writer.markdown(VERSION, commits, {})
+        .then(function (changelog) {
+          return changelog.split('\n');
+        })
+        .filter(function (line) {
+          return line.indexOf(hash) > -1;
+        })
+        .get(0)
+        .then(function (line) {
+          var regex = new RegExp('^\\* other changes \\(1234567a\\)$');
+
+          Expect(line).to.match(regex);
```

Comment:
Since this `regex` is just checking that it equals a literal string, you can just use `.to.eql` instead:

```js
Expect(line).to.eql('* other changes (1234567)');
```